## Cost Benefit Analysis

Conduct a cost-benefit analysis. This should include annual cost projections for various levels of pesticide coverage (cost) and the effect of these various levels of pesticide coverage (benefit). (Hint: How would we quantify the benefit of pesticide spraying? To get "maximum benefit," what does that look like and how much does that cost? What if we cover less and therefore get a lower level of benefit?)

To determine the cost effectiveness of spraying, we first need to consider the following:<br>
**Direct costs**: These would mainly include the cost of procuring chemicals required for spraying, inpatient and outpatient costs <br>
**Indirect costs**: These would be productivity/economic costs incurred from people seeking short/long term treatment for the west nile virus. <br>


### Cost of procuring chemicals for spraying

From [external research](https://www.chicago.gov/city/en/depts/cdph/provdrs/healthy_communities/news/2020/august/city-to-spray-insecticide-thursday-to-kill-mosquitoes0.html), we know a few facts about sprays: <br>
1) spraying is usually conducted around 8pm till 1 am using trucks with low volume sprays, and the EPA approved spray is Zenivex. <br>
2) the spray rate is [1.5 ounces per acre](https://www.nbcchicago.com/news/local/city-to-spray-mosquito-killing-insecticide-wednesday-night/64951/). <br>
3) the cost per gallon (128 ounces) of Zenivex is [approximately \$ 300 USD.](http://www.gfmosquito.com/wp-content/uploads/2017/07/2017-ND-Mosq.-Control-Quotes-Tabulation.pdf) <br>
4) the city of chicago is approximately 145,300 acres in [total area](https://www.themeasureofthings.com/results.php?comp=area&unit=a&amt=160000&p=1). <br>

Therefore, in order to achieve "maximum" benefit, let's assume that the Chicago Department of Public Health (CDPH) adopts an aggressive approach towards culling the virus, and decides to conduct spraying on the whole of Chicago.

In [1]:
# Cost of procuring Zenivex to spray entire Chicago

# amount of spray (ounces) required to spray the whole of Chicago
spray_amt = 145300 / 1.5

#amount of spray in gallons
spray_gallon = spray_amt / 128

#costs of procuring the spray
spray_cost = spray_gallon * 300

spray_cost

227031.25

It costs the state \$227031 to purchase Zenivex for one spraying exercise for the whole of Chicago. Considering the maximum benefit scenario, we now assume that the CDPH takes into account the incubation period of mosquitos, and decides to do a bi-weekly spraying every summer for the whole of Chicago.

In [2]:
# Annual costs of spraying

annual_spray_cost = spray_cost * 6 

annual_spray_cost

1362187.5

In the maximum benefit scenario, the CDPH will need to set aside a budget of approximately **\$1.36 million** annually to conduct spraying for the whole of Chicago.